homeassistant/components/blastbot_cloud/__init__.py

```diff
@@ -0,0 +1,87 @@
+"""The Blastbot Cloud integration."""
+import asyncio
+
+from blastbot_cloud_api.api import BlastbotCloudAPI
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME, EVENT_HOMEASSISTANT_STOP
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_validation as cv
+
+from .const import DOMAIN, LOGGER
+
+CONFIG_SCHEMA = vol.Schema(
+    {
+        DOMAIN: vol.Schema(
+            {
+                vol.Required(CONF_USERNAME): cv.string,
+                vol.Required(CONF_PASSWORD): cv.string,
+            }
+        )
+    },
+    extra=vol.ALLOW_EXTRA,
+)
+
+
+# List the platforms that you want to support.
+PLATFORMS = ["switch", "climate", "remote"]
```

Comment:
Please limit your PR to a single platform. For more information see:

<https://developers.home-assistant.io/docs/creating_component_code_review#5-make-your-pull-request-as-small-as-possible>

Reply:
one question: is it ok to keep the other platforms' files but only enable one? or should I remove the files too for now?

Reply:
Please remove the code for now. You can save the code in another local branch. Then when this PR is merged it's easy to open a PR per platform to add.